packages/flutter_tools/test/integration.shard/test_utils.dart

```diff
@@ -46,7 +46,12 @@ Future<void> getPackages(String folder) async {
     'pub',
     'get',
   ];
-  final ProcessResult result = await processManager.run(command, workingDirectory: folder);
+  final ProcessResult result = await processManager.run(command,
+    workingDirectory: folder,
+    environment: <String, String>{
+    'FLUTTER_INTEGRATION_TEST': 'true'
```

Comment:
In the previous two instances of this it's set to 'test' rather than 'true'.

Reply:
Fixed